docs/full/README.md

```diff
@@ -158,27 +158,29 @@ variables.  These variables can be set in the shell or using `tanzu config set e
+| `TANZU_CLI_SHOW_TELEMETRY_CONSOLE_LOGS`                             | Print telemetry logs (defaults to off).                                                                                                                                                                                                                                                                        | `1` or `true` to print, `0`, `false`, `""` or unset not to print                                                                                               |
+| `TANZU_CLI_SKIP_TAP_SCOPES_VALIDATION_ON_TANZU_CONTEXT`             | If set, CLI would skip TAP scopes validation on `tanzu` type context created using `tanzu login` or `tanzu context create` command.                                                                                                                                                                            | `1`, `true` to skip, `0`, `false`, `""` or unset to allow TAP scopes validation                                                                                |
+| `TANZU_CLI_SKIP_UPDATE_KUBECONFIG_ON_CONTEXT_USE`                   | Do not synchronize the active Kubernetes context when the Tanzu context is changed.                                                                                                                                                                                                                            | `1` or `true` to skip, `0`, `false`, `""` or unset to do the synchronization                                                                                   |
+| `TANZU_CLI_SUPPRESS_SKIP_SIGNATURE_VERIFICATION_WARNING`            | Suppress the warning message that some plugin discoveries are not being verified due to the use of `TANZU_CLI_PLUGIN_DISCOVERY_IMAGE_ SIGNATURE_VERIFICATION_SKIP_LIST`.  The use of this variable should be avoided as it can put your environment at risk.                                                   | `1`, `true` to suppress, `0`, `false`, `""` or unset to allow the message                                                                                      |
+| `TANZU_CLI_USE_STABLE_KUBE_CONTEXT_NAME`                            | If set, CLI would use stable context name and not update the kubecontext name when user updates the tanzu context active resource using `tanzu project use`, `tanzu space use` and `tanzu clustergroup use` commands.                                                                                          | `1`, `true` to allow using stable kubecontext name, `0`, `false`, `""` or unset to not use stable kubecontext name                                             |
```

Comment:
nit: "would use stable context name and" -> will
stable context name is implied by the second part of the sentence, and "stable context name" is unclear because it could refer to CLI or kube context (it is the latter)

Reply:
Thank you! updated.